reacthookspring/webclient/app/src/app/assets/themes/MainTheme.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import {ThemeProvider} from "@mui/material";
+import ComponentTheme from "../../assets/themes/ComponentTheme";
```

Comment:
Abgesehen davon, dass ich nicht glaube, dass das so funktioniert. Wieso ist die MainTheme.jsx in die Assets gewandert. Was ist der Unterschied zum AppHeader.jsx. Mich würden einfach die Gedanken dahinter interessieren